LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
ähm, the license is not an "unnecessary file".
Your first commit does not seem right.
Please try to do the following steps:

```
git checkout master
git pull
git checkout DubbleClick-patch-1
git rebase -i master
```

And remove your [last commit](6cf8cd0bd00f9ca61d1350be19386f2fb7e2bb1b) (skip) from the rebase.
Also please skip all commits, that are already part of master (all travis related commits) and then fix any remaining conflicts, if any (should all be fine).